tests/e2e/support/objects/app-admin-settings/users/index.ts

```diff
@@ -150,7 +160,17 @@ export class Users {
     email: string
     password: string
   }): Promise<void> {
-    await createUser({ page: this.#page, name, displayname, email, password })
+    const response = await createUser({ page: this.#page, name, displayname, email, password })
+    this.#usersEnvironment.storeUser({
+      key: name,
+      user: {
+        id: response['onPremisesSamAccountName'],
+        displayName: response['displayName'],
+        password: password,
+        email: response['mail'],
+        uuid: response['id']
+      }
```

Comment:
why not using response.xyz?

Reply:
or storeUser(response)

Reply:
> why not using response.xyz? 
response type is dynamic so we can't use the dot operator. 

> or storeUser(response)
API response doesn't contain password init which we need for login/re-login purpose